vyper/venom/passes/simplify_cfg.py

```diff
@@ -0,0 +1,82 @@
+from vyper.utils import OrderedSet
+from vyper.venom.basicblock import IRBasicBlock
+from vyper.venom.function import IRFunction
+from vyper.venom.passes.base_pass import IRPass
+
+
+class SimplifyCFGPass(IRPass):
+    visited: OrderedSet
+
+    def _merge_blocks(self, a: IRBasicBlock, b: IRBasicBlock):
+        a.instructions.pop()
```

Comment:
i think this can be simplified right? like 
```python
a_phis = [s for s in a.instructions if s.opcode == "phi"]
b_phis = [s for s in b.instructions if s.opcode == "phi"]
a_insts = [s for s in a.instructions if s.opcode != "phi"]
b_insts = [s for s in b.instructions if s.opcode != "phi"]
b.instructions = a_phis + b_phis + a_insts + b_insts
```

Reply:
I think just iterating over the second block once is clearer and faster than doing multiple iterations and conditions here. I vote to keep it as is.